domains/simeon.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "My Portfolio website",
+  "repo": "https://github.com/Nwachukwuchinedu/simeon-portfolio",
+  "owner": {
+    "username": "Nwachukwuchinedu",
+    "email": "chinedu.simoen2020@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://nwachukwuchinedu.github.io/simeon-portfolio/"
```

Cannot access website, 404 not found
also, CNAME can't contain protocol or `/`
```suggestion
    "CNAME": "nwachukwuchinedu.github.io"
```